### ISSUE TITLE
Double-mapped term shows up only once in term frequency

### ISSUE THREAD

**Issue by thomasstjerne** [gbif/gbif-data-validator#58]
From @mdoering from skype 
- when validating an archive that maps the same column to multiple terms there only one gets displayed in the frequency table. Likely this is a bug in the validator backend? see missing dwc:scientificName in the taxon core here: https://www.gbif.org/tools/data-validator/1508848348033 vs meta mapping here https://github.com/mdoering/zeigerwerte/blob/master/meta.xml#L8

**Comment by cgendreau** [comment]
No, that won't work and will require some refactoring (some functions use associative arrays).
